mesonbuild/modules/python3.py

```diff
@@ -17,13 +17,15 @@
 import typing as T
 
 from .. import mesonlib
-from . import ExtensionModule, ModuleInfo
+from . import ExtensionModule, ModuleInfo, ModuleState
+from ..build import CustomTarget, CustomTargetIndex, BuildTarget, GeneratedList, ExtractedObjects, StructuredSources, SharedModule
 from ..interpreter.type_checking import SHARED_MOD_KWS
 from ..interpreterbase import typed_kwargs, typed_pos_args, noPosargs, noKwargs, permittedKwargs
 from ..build import known_shmod_kwargs
```

Comment:
Might as well sort this and merge both imports from `..build` while we are at it.